### ISSUE TITLE
Update to Elm 1.8

### ISSUE THREAD

**Issue by SamuelBambu** [ThomasWeiser/elmfire#17]
This library has not been updated for Elm 1.8, it would be nice to have that and use it with current projects running on Elm 1.8

**Comment by andykingking** [comment]
Agreed. @ThomasWeiser If there is a todo list I would be happy to help out.

Derp. Just saw this: https://github.com/ThomasWeiser/elmfire/pull/16
